tools/syscount.py

```diff
@@ -0,0 +1,500 @@
+#!/usr/bin/env python
+#
+# syscount   Summarize syscall counts and latencies.
+#
+# USAGE: syscount [-p PID] [-i INTERVAL] [-T TOP] [-x] [-L] [-m] [-P] [-l]
+#
+# Copyright 2017, Sasha Goldshtein.
+# Licensed under the Apache License, Version 2.0 (the "License")
+#
+# 15-Feb-2017   Sasha Goldshtein    Created this.
+
+from bcc import BPF
+from itertools import izip_longest
+from time import sleep, strftime
+import argparse
+import sys
+
+#
+# Syscall table for Linux x86_64, not very recent.
+# Automatically generated from strace/linux/x86_64/syscallent.h using the
+# following command:
+#
+#  cat syscallent.h | awk -F, '{ gsub(/[ \t"}]/, "", $4);
+#                                gsub(/[ \t/*]/, "", $5);
+#                                print "    "$5": \""$4"\","; }
+#                              BEGIN { print "syscalls = {" } END { print "}" }'
+#
```

Comment:
belated comment, but yes, thanks for including the regeneration command. Anytime I have a hardcoded table, I put the source commands in there too.